PortfolioApp/Model/UsersModels/UsersModels.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+struct User: Decodable {
```

Comment:
The file should name as struct name. Don't invent custom names. Files name should be equal to model name